doc/1.manual/x2.htm

```diff
@@ -447,9 +447,10 @@ <H3> <A id="s2.7">2.2.7. Context menu for 'Properties', 'Open' and 'Help'</A> </
 
 <P>All the "clicking" mentioned above is done with the left mouse button.
 The right button, instead, opens a context menu offering "Properties",
-"Open" and "Help" (either in edit or run mode). For Apple users who may
-only have one button on their mouse, double-clicking or <kbd>Alt + Click</kbd>
-are mapped to right-click and on trackpads you can click with two fingers.
+"Open" and "Help" (either in edit or run mode). Apple users, who may
+only have one button should know that <kbd>Control + Click</kbd> is mapped
```

Comment:
minor nitpick, but think the [preferred form](https://www.w3schools.com/Tags/tag_kbd.asp) for key-combinations is `<kbd>Control</kbd> + <kbd>Click</kbd>`  (that is: <kbd>Control</kbd>+<kbd>Click</kbd> rather than <kbd>Control + Click</kbd>).

it should of course be consistent across all pages.

Reply:
agreed that this is what `<kbd>` was intended for, i assume. :) ... although the current version of a "key combination highlighting" is also quite readable.

@porres : if you agree with this change, there is https://github.com/ben-wes/pure-data/commit/a270dae9ed1b71eca2756e0bb559b5324a5d9b81

Reply:
I likr it, send me a PR please